changedetectionio/notification.py

```diff
@@ -101,7 +101,9 @@ def process_notification(n_object, datastore):
                 apobj.notify(
                     title=n_title,
                     body=n_body,
-                    body_format=n_format)
+                    body_format=n_format,
+                    attach=None if not n_object['screenshot'] else n_object['screenshot']
```

Comment:
```suggestion
                    # False is not an option for AppRise, must be type None
                    attach=None if not n_object.get('screenshot') else n_object.get('screenshot')
```

Reply:
it's good to put that info in there to save other developers a lot of wasted time in the future :)